src/psyclone/tests/psyir/nodes/reference_test.py

```diff
@@ -182,7 +251,7 @@ def test_array_create_invalid():
     with pytest.raises(GenerationError) as excinfo:
         _ = Array.create([], [])
     assert ("name argument in create method of Array class should "
```

Comment:
Is it now the "symbol argument" rather than "name"?

Reply:
Yes, changed. Also removed spurious broken test that is no longer valid.